jwst/undersampling_correction/tests/test_undersampling_correction.py

```diff
@@ -0,0 +1,145 @@
+import numpy as np
+
+from jwst.datamodels import RampModel
```

Comment:
I think these imports will still work, because they'll be automatically redirected to stdatamodels.jwst.datamodels, but we might as well go ahead and change them to point directly to the new location, as has been done in all the step codes.

Reply:
No errors were given when I did the rebase:
 (jwst_h) sal:jwst grumm$ git rebase  upstream/master 
Successfully rebased and updated refs/heads/master.

 And yes, I also thought that the jwst.datamodels would have been removed. I will look into this.